docs/source/faq.rst

```diff
@@ -1,6 +1,16 @@
 FAQ
 ===
 
+- :ref:`Can I use Optuna with X? (where X is your favorite ML library)`
```

Comment:
I'm not 100% sure, but we might be able to use `..contents::` directive to generate the same HTML with fewer maintenance cost.
```
.. contents::
   :local:
```

Reply:
That definitely seems like the right way to go. Thanks, applied!

Reply:
Just leaving a comment about the directive. https://stackoverflow.com/questions/24129481/how-to-include-a-local-table-of-contents-into-sphinx-doc